### PR TITLE
Update website example to match latest Fe

### PR DESCRIPTION
**Pull request by cburgdorf**
### To-Do

[//]: # (Stay ahead of things, add list items here!)
- [ ] OPTIONAL: Update [Spec](https://github.com/ethereum/fe/blob/master/docs/src/spec/index.md) if applicable
- [ ] Add entry to the [release notes](https://github.com/ethereum/fe/blob/master/newsfragments/README.md) (may forgo for trivial changes)

- [ ] Clean up commit history

### CODE REVIEW

File: docs/src/spec/items/events.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Events
```
Comment (Y-Nak) [inline docs/src/spec/items/events.md:1]: Is this a leftover?

File: docs/src/spec/type_system/types/event.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Event Types
```
Comment (Y-Nak) [inline docs/src/spec/type_system/types/event.md:1]: ditto

Comment (cburgdorf) [reply docs/src/spec/type_system/types/event.md:1]: Ah, those are auto generated

**Comment by Y-Nak** [comment]
👍

**Pull request merged**